tests/behat/behat_filter_imageopt.php

```diff
@@ -28,12 +29,15 @@
 
 /**
  * Imageopt filter context
+ * @package   filter_imageopt
  * @author    Guy Thomas <brudinie@gmail.com>
  * @copyright Copyright (c) 2017 Guy Thomas.
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 class behat_filter_imageopt extends behat_base {
     /**
+     * If image optimiser is eneabled.
```

Comment:
Note - you don't need to add additional doc block comments when the @Given / @When / @Then is clear enough.